study.md

```diff
@@ -40,14 +51,21 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+The exclamation point indicates that the method will change the object which it's called on.  In the situation above, it indicates that it will modify "my_string"
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+class Home
```

Comment:
This defines a class but doesn't instantiate an instance of it.  `home = Home.new(...)`.